ObjectPool-Tests/OPBasicPoolTest.class.st

```diff
@@ -153,18 +153,21 @@ OPBasicPoolTest >> testMigrationObjectsToAnotherPool [
 OPBasicPoolTest >> testMigrationObjectsToAnotherPoolWhenBorrowedObjectsExists [
 
 	| anotherPool |
+
 	pool objectToPool: #object1.
 	pool objectToPool: #object2.
 	pool borrow.
-	
+
 	anotherPool := self poolClass new.
-	
-	[pool migrateObjectsInto: anotherPool andDo: nil.
-	self assert: false description: 'should raise error'
-	] ifError: [].
-		
-	self assert: anotherPool numberOfAvailableObjects = 0.
-	self assert: pool numberOfAvailableObjects = 2
+
+	self
+		should: [ 
+			pool migrateObjectsInto: anotherPool andDo: nil.
+			self assert: false description: 'should raise error'
+			]
+		raise: Error.
```

Comment:
I wonder if it's not more explicit to say it shouldn't raise AssertionFailure :)

Reply:
I think it will fail, because `migrateObjectsInto: anotherPool andDo:` is raising an `Error` and that is the part failing.